compiler/AST/expr.cpp

```diff
@@ -1477,8 +1478,8 @@ GenRet codegenFieldPtr(
     base = codegenValue(base);
   } else {
     // not a class. base is a lvalue pointer.
-    if( !fLLVMWideOpt ) INT_ASSERT(base.isLVPtr == GEN_PTR);
-    else INT_ASSERT(base.isLVPtr != GEN_VAL);
+    //if( !fLLVMWideOpt ) INT_ASSERT(base.isLVPtr == GEN_PTR);
+    //else INT_ASSERT(base.isLVPtr != GEN_VAL);
```

Comment:
I'm starting to wonder if you just commented out all asserts during development? Could these be turned back on?